frontend/src/bundles/auth/store/actions.ts

```diff
@@ -5,17 +5,20 @@ import {
     type UserSignUpRequestDto,
     type UserSignUpResponseDto,
 } from '~/bundles/users/users.js';
+import { StorageKey } from '~/framework/storage/storage.js';
 
 import { name as sliceName } from './slice.js';
 
 const signUp = createAsyncThunk<
     UserSignUpResponseDto,
     UserSignUpRequestDto,
     AsyncThunkConfig
->(`${sliceName}/sign-up`, (registerPayload, { extra }) => {
+>(`${sliceName}/sign-up`, async (registerPayload, { extra }) => {
     const { authApi } = extra;
 
-    return authApi.signUp(registerPayload);
+    const data = await authApi.signUp(registerPayload);
+    authApi.Storage.set(StorageKey.TOKEN, data.token);
```

Comment:
Use storage from extra args

```suggestion
    const data = await authApi.signUp(registerPayload);
    storage.set(StorageKey.TOKEN, data.token);
```